client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -29,6 +29,8 @@ export class CommentAuthor extends Component {
 		isPostView: PropTypes.bool,
 	};
 
+	shouldComponentUpdate = nextProps => ! isEqual( this.props, nextProps );
```

Comment:
this worked well for me on this component.

```js
shouldComponentUpdate = () => false;
```

Reply:
Probably a bit too much! 😄 

This is what happens on first load, without cached data:

<img width="489" alt="screen shot 2017-11-22 at 12 32 36" src="https://user-images.githubusercontent.com/2070010/33127719-99cbbf0a-cf81-11e7-9995-813d3dfed1ef.png">

`CommentAuthor` loses the gravatar and defaults to "Anonymous" instead of "coponsa8c" as shown in `CommentAuthorMoreInfo`, which updates with the fresh data.

Reply:
maybe I never hit the _clear cache for the funsies_ status.